.bibtoolrsc

```diff
@@ -38,6 +38,7 @@ preserve.key.case = on
 sort.cased = off
 print.use.tab = off
 
+delete.field = { abstract }
```

Comment:
Unrelated change, doesn't seem to have an effect? I don't mind it, but it'd be good to mention it in the commit message